qa/tasks/mgr/dashboard/test_rbd.py

```diff
@@ -844,14 +844,20 @@ def test_purge_trash(self):
         time.sleep(1)
 
         self.purge_trash('rbd')
-        self.assertStatus(200)
+        self.assertStatus([200, 201])
 
         time.sleep(1)
 
         trash_not_expired = self.get_trash('rbd', id_not_expired)
         self.assertIsNotNone(trash_not_expired)
 
-        trash_expired = self.get_trash('rbd', id_expired)
+        # A wait loop to tolerate delayed-purge on busy cluster
+        retries = 6
```

Comment:
You could also use the already existing helper method `wait_until_equal` to perform the delay.
```suggestion
        self.wait_until_equal(self.get_trash('rbd', id_expired), None, 60)
```

Reply:
Thanks, it's handy. I just need to also make the call anonymous with lambda.